README.md

```diff
@@ -65,7 +65,8 @@ With the 11 bit aligned implementation we have:
 
 ORC is not supported (See [Issue #4](https://github.com/shayanhabibi/loony/issues/4))
 
-Spawn (Nims `threadpool` module) is not supported. *It won't ever be; don't use spawn unless you are a masochist in general.*
+Spawn (via Nim's `threadpool` module) is not supported. *It won't ever be; don't use spawn unless you are a masochist in general.*
```

Comment:
I'm not sure why this would be the case; I think it'd make more sense to wait for a bug report so we can understand the problem better, and ensure we have a canary to show when it suddenly starts working.

Reply:
I just modified the existing sentence. Should I remove it?

Reply:
Sure.